students/Scott_Hancock/session04/mailroom/mailroom.bak

```diff
@@ -0,0 +1,119 @@
+class Donor:
+    """ Class to keep track of donors with a donor name and list of donation amounts """
+
+    def __init__(self, name=''):
+        """ """
+        self.name = name
+        self.donationList = list()
+
+    def addDonation(self, amount):
+        """ Add a donation amount to the list """
+        self.donationList.append(amount)
+
```

Comment:
You could add some testing here to ensure that amount is an int or float... this would get weird if it was a string.